spec/unit/azure_server_create_spec.rb

```diff
@@ -140,76 +140,81 @@ def test_params(testxml, chef_config, role_name, host_name)
       end
     end
 
-    context "server create options" do
-      before do
-        Chef::Config[:knife][:bootstrap_protocol] = 'ssh'
-        Chef::Config[:knife][:ssh_password] = 'ssh_password'
-        Chef::Config[:knife][:ssh_user] = 'ssh_user'
-        Chef::Config[:knife].delete(:azure_vm_name)
-        Chef::Config[:knife].delete(:azure_storage_account)
-        @bootstrap = Chef::Knife::Bootstrap.new
-        allow(Chef::Knife::Bootstrap).to receive(:new).and_return(@bootstrap)
-        expect(@bootstrap).to receive(:run)
-        allow(@server_instance).to receive(:msg_server_summary)
-      end
-
-      it "quick create" do
-        expect(@server_instance).to receive(:is_image_windows?).at_least(:twice).and_return(false)
-        Chef::Config[:knife][:azure_dns_name] = 'vmname' # service name to be used as vm name
-        expect(@server_instance).to receive(:get_dns_name)
-        @server_instance.run
-        expect(@server_instance.config[:azure_vm_name]).to be == "vmname"
-        testxml = Nokogiri::XML(@receivedXML)
-        expect(xml_content(testxml, 'MediaLink')).to_not be nil
-        expect(xml_content(testxml, 'DiskName')).to_not be nil
-        test_params(testxml, Chef::Config[:knife], Chef::Config[:knife][:azure_dns_name],
-        Chef::Config[:knife][:azure_dns_name])
-      end
-
-      it "quick create with wirm - API check" do
-        expect(@server_instance).to receive(:is_image_windows?).at_least(:twice).and_return(true)
-        Chef::Config[:knife][:azure_dns_name] = 'vmname' # service name to be used as vm name
-        Chef::Config[:knife][:winrm_user] = 'opscodechef'
-        Chef::Config[:knife][:winrm_password] = 'Opscode123'
-        Chef::Config[:knife][:bootstrap_protocol] = 'winrm'
-        expect(@server_instance).to receive(:get_dns_name)
-        @server_instance.run
-        expect(@server_instance.config[:azure_vm_name]).to be == "vmname"
-        testxml = Nokogiri::XML(@receivedXML)
-        expect(xml_content(testxml, 'WinRM')).to_not be nil
-        expect(xml_content(testxml, 'Listeners')).to_not be nil
-        expect(xml_content(testxml, 'Listener')).to_not be nil
-        expect(xml_content(testxml, 'Protocol')).to be == "Http"
-      end
-
-      it "generate unique OS DiskName" do
-        os_disks = []
-        allow(@bootstrap).to receive(:run)
-        allow(@server_instance).to receive(:validate!)
-        Chef::Config[:knife][:azure_dns_name] = 'vmname'
-
-        5.times do
-          @server_instance.run
-          testxml = Nokogiri::XML(@receivedXML)
-          disklink = xml_content(testxml, 'MediaLink')
-          expect(os_disks).to_not include(disklink)
-          os_disks.push(disklink)
+	context "timeout parameters" do
+	
+      it "uses correct values when not specified" do
```

Comment:
Removed wrong part during conflict resolved!!!

Above specs uses **expect**  and **allow**,  instead of **should** and **stub** respectively, which is deprecated in new rspec version!